src/compiler/types.ts

```diff
@@ -2691,6 +2691,8 @@ namespace ts {
         AddUndefined                    = 1 << 13,  // Add undefined to types of initialized, non-optional parameters
         WriteClassExpressionAsTypeLiteral = 1 << 14, // Write a type literal instead of (Anonymous class)
         InArrayType                     = 1 << 15,  // Writing an array element type
+        UseAliasEvenIfNotAccessible     = 1 << 16,  // For a `type T = ... ` defined in a different file, write `T` instead of its value,

```

Comment:
This seems suspiciously correlated to `UseTypeAliasValue`. I'm not sure what the right way to emphasize the difference is, but it seems like a smell.